server/server_test.go

```diff
@@ -119,8 +119,8 @@ func (s *ServerTestSuite) TestPush() {
 func (s *ServerTestSuite) TestReviewTimeout() {
 	require := s.Require()
 
-	reviewSleep, _ := time.ParseDuration("5ms")
-	reviewTimeout, _ := time.ParseDuration("1ms")
+	reviewSleep := 1 * time.Second
```

Comment:
that is very huge timeout. I understand 5ms can be too small but for example 100ms or 200ms?
This change slows down tests a lot which isn't cool when you use them to validate the code...